config.example.json

```diff
@@ -5,5 +5,6 @@
   "accessSecret": "TWITTER ACCESS SECRET",
   "slackWebhook": "SLACK INCOMING WEBHOOK",
   "slackChannel": "OPTIONAL SLACK CHANNEL TO SEND TO",
-  "keywords": "keywords,your want,to track"
+  "keywords": "keywords,your want,to track",
+  "keywordsIgnored": "keywords,you do not want,to track"
```

Comment:
I'll correct that.

@calavera I remembered why I did this - we're not touching the keywords string (it's consumed by Twitter's api), and I didn't want to have two different signatures for `keywords` and `keywordsIgnored`. The related code is almost negligible in size.